src/scripts/client.js

```diff
@@ -171,26 +175,28 @@
     bannerWasShown = true;
   }
 
-  function checkIt() {
-    var wayback_page_url = window.__WAYBACK_PAGE_URL;
-    var wayback_response = window.__WAYBACK_RESPONSE;
-    if (wayback_page_url && wayback_response) {
-      // Some pages use javascript to update the dom so poll to ensure
-      // the banner gets recreated if it is deleted.
-      enforceBannerInterval = setInterval(function() {
-        createBanner(wayback_response.archived_snapshots.closest.url);
-      }, 500);
+  function checkIt(wayback_url) {
+    // Some pages use javascript to update the dom so poll to ensure
+    // the banner gets recreated if it is deleted.
+    enforceBannerInterval = setInterval(function() {
```

Comment:
It's generally considered good practice to clean up after yourself. It also prevents memory leaks, although I think if the page object in memory dies, the timer would die as well.